static/news.ru.html

```diff
@@ -1,12 +1,29 @@
 <h1 id="helpPage_newsTitle">Новости</h1>
 
+<h2 class="title">Новый режим, регулировка сложности и не только</h2>
+<span class="date">
+    __.08.2020
+</span><span class="version">
+    v1.2
+</span><span class="new">
+    * Новое
+</span>
+<p>
+Играть стало ещё удобнее и интереснее!
+
+<li>Добавили режим игры со словами игроков: каждый участник пишет слова, из которых формируется набор слов в шляпе</li>
+<li>Появились словари разного уровня сложности на русском языке</li>
+<li>Если у вас есть собственный словарь, теперь вы сможете его загрузить и играть со словами из него</li>
+<li>Появилась возможность ограничить не количество слов, а количество кругов</li>
+<li> Отображение количества ходов до вашего следующего участия. Захотели налить чайку? Теперь всё можно спланировать!</li>
+<li>Общая ссылка на новую игру. После окончания старой игры новая комната создастся автоматически</li>
```

Comment:
Предлагаю чуть-чуть подробностей, чтобы было понятно. А то непонятно, что за комната, как в неё входить и т.п.
```suggestion
<li>Общая ссылка на новую игру. После окончания старой игры новая комната создастся автоматически. Нажмите "играть ещё раз", чтобы снова сыграть всем вместе</li>
```

Reply:
Формально же в новую игру не все могут прийти. Такое описание точно сойдёт?

Reply:
Я так-то очень даже за твой вариант, просто уточняю

Reply:
Ну я хоть какое-то уточнение даю. Формально самого рвёт на части, но для наглядности сойдёт. Так сказать, на месте разберутся, хотя лучше ещё обновить FAQ.

Reply:
Окей, тогда одобряю